template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/deployment/batch_inference/notebooks/BatchInference.py.tmpl

```diff
@@ -34,7 +34,7 @@ dbutils.widgets.text(
 {{else}}
 # Unity Catalog registered model name to use for the trained mode.
 dbutils.widgets.text(
-    "model_name", "dev.{{template `schema_name` .}}.{{template `model_name` .}}", label="Model Name"
+    "model_name", "dev.{{template `schema_name` .}}.{{template `model_name` .}}", label="Full (Three-Level) Model Name"
```

Comment:
The variable description looks good. I'm more referring to the `model_name` variable name. We can also wait and see if there's customer feedback about the name.

Reply:
Yeah but then we'd have to add an `if/else` condition around every time that `model_name` is used on whether or not UC Models is enabled, and I didn't want to risk missing one of those this close to public preview.